lib/form/html.js

```diff
@@ -90,6 +90,7 @@ const addRequired = (html, input, index) => {
 
     if (input.required === 'save' || inp.required === 'save' || input.required === 'publish' || inp.required === 'publish') {
       let level = '';
+      const required = input.required && inp.required;
```

Comment:
@ineedsubstance did you add this line instead of the conditional?

```
const level = `required--${required}`;
```
if you removed the conditional without adding that line, then it would break the test, yes.